<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">   background-color: #0D1A4A;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Song Name</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 8px;
    background: #e0e0e0;
    border-radius: 4px;
    margin: 2px 0;
}

QSlider::sub-page:horizontal {
    background: #1565C0;
    border-radius: 4px;
}

QSlider::add-page:horizontal {
    background: #e0e0e0;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: white;
    border: 3px solid #2196F3;
    width: 24px;
    height: 24px;
    border-radius: 12px;
    margin: -8px 0;
}

QSlider::handle:horizontal:hover {
    background: white;
    border: 3px solid #1976D2;
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_gofront">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    border: 2px solid #2196F3;
    border-radius: 25px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 50px;
    max-width: 50px;
    min-height: 50px;
    max-height: 50px;
    padding: 0px;
}

QPushButton:hover
{
    border: 2px solid #1976D2;
    background-color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_getback">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    border: 2px solid #2196F3;
    border-radius: 25px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 50px;
    max-width: 50px;
    min-height: 50px;
    max-height: 50px;
    padding: 0px;
}

QPushButton:hover
{
    border: 2px solid #1976D2;
    background-color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_nxtsong">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    border: 2px solid #2196F3;
    border-radius: 25px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 50px;
    max-width: 50px;
    min-height: 50px;
    max-height: 50px;
    padding: 0px;
}

QPushButton:hover
{
    border: 2px solid #1976D2;
    background-color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_lastsong">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    border: 2px solid #2196F3;
    border-radius: 25px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 50px;
    max-width: 50px;
    min-height: 50px;
    max-height: 50px;
    padding: 0px;
}

QPushButton:hover
{
    border: 2px solid #1976D2;
    background-color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    border: 2px solid #2196F3;
    border-radius: 30px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 60px;
    max-width: 60px;
    min-height: 60px;
    max-height: 60px;
    padding: 0px;
}

QPushButton:hover
{
    border: 2px solid #1976D2;
    background-color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_soundoff">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>250</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>160</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open </string>
    </property>
    <addaction name="actionOpen_file"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
